mantidproject/mantid
# New loader for ILL diffractometers

[files.zip](https://github.com/mantidproject/mantid/files/1014709/files.zip)

**To test:**
1. Set ILL/D20 as instrument.
2. Load a file from the attachment with `LoadILLDiffraction`
3. Check the output workspace
 - Should have 1 column
 - First row should be marked as monitor
 - Errors should be present (square root of the count)
 - Instrument should be attached (check from instrument viewer)
 - Check that the sample logs exist
4. Load the same file with the generic `Load`, output should be the same

<strike>**Do not merge before PR #18453**</strike>



Fixes #18868 .



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
